srcpkgs/textadept/template

```diff
@@ -1,87 +1,79 @@
- 48d66576051b6c78388faad09b70493093264588fcd0f258ddaab1cdd4a15ffe
+ https://www.scintilla.org/scintilla${_scintilla_version}.tgz
+ https://github.com/orbitalquark/scinterm/archive/scinterm_${_scinterm_version}.zip
+ https://github.com/orbitalquark/scintillua/archive/scintillua_${_scintillua_version}.zip
+ https://www.lua.org/ftp/lua-${_lua_version}.tar.gz
+ https://www.inf.puc-rio.br/~roberto/lpeg/lpeg-${_lpeg_version}.tar.gz
+ https://github.com/keplerproject/luafilesystem/archive/v${_luafilesystem_version}.zip
+ https://github.com/orbitalquark/lua-std-regex/archive/${_luastdregex_version}.zip
+ https://github.com/ThomasDickey/cdk-snapshots/archive/refs/tags/${_cdksnapshots_version}.tar.gz
+ https://www.leonerd.org.uk/code/libtermkey/libtermkey-${_libtermkey_version}.tar.gz
+ https://github.com/DaanDeMeyer/reproc/archive/refs/tags/v${_reproc_version}.zip"
+checksum="2e45cbd492f88328cb9a91495af2885a1e7d72b9c33ddd1eeb43c1f23f7b7bb2
+ 7994a854a3012269f35ecc2cb725b8f7e209e410dc96fe141c9a15ccfebd41fc
+ 1a78abc9e1377c28d323af62fe7c4376f1eb3007adbbfa6d3e1e3eaece703666
+ fa69920c9b8413a25a75f6661db3de2e4ed3f7306143367137fbbd1e3bc480e2
+ 9fbf5e28ef86c69858f6d3d34eccc32e911c1a28b4120ff3e84aaa70cfbf1e30
+ 4b155d67d2246c1ffa7ad7bc466c1ea899bbc40fef0257cc9c03cecbaed4352a
  e3a6beca7a8a90522eed31db6ccdc5ed65a433826500c6862784e27671b9e18a
- 093a4f973196083610c4489e8d9272f340ebf82d48c064f1542c6464eda2af82
- 007f5de880cb2eebd8556df7e4cd8673d5e64c9970147eee6923a814c29faaed
- 6c0d87c94ab9915e76ecd313baec08dedf3bd56de83743d9aa923a081935d2f5"
-skip_extraction="scintilla445.tgz
- 6a774158d8a3c7bc7ea120bc01cdb016fa351a7e.zip
- scintillua_4.4.5-2.zip
- lua-5.3.5.tar.gz
- lpeg-1.0.2.tar.gz
- v1_8_0.zip
- 64587546482a1a6324706d75c80b77d2f87118a4.zip
- cdk-5.0-20200923.tgz
- libtermkey-0.20.tar.gz"
+ 5b684a1ce7ea632a37aa4f98bcf265cd97d9d70c5998c56985295c6aa97e74e1
+ 69a2c96ce23b3137929ecc011c67b030554cc532ab00a011cbbccf01dacaa611
+ 6945bd3c4aaa83da83d80a045c5563da4edd7d0374c62c0d35aec09eb3014600
+ 21b168c38adb22f4b48391d8193dbc3927f1f2e678c2058d6944a29b64acc6c9"
+skip_extraction="scintilla${_scintilla_version}.tgz
+ scinterm_${_scinterm_version}.zip
+ scintillua_${_scintillua_version}.zip
+ lua-${_lua_version}.tar.gz
+ lpeg-${_lpeg_version}.tar.gz
+ v${_luafilesystem_version}.zip
+ ${_luastdregex_version}.zip
+ ${_cdksnapshots_version}.tar.gz
+ libtermkey-${_libtermkey_version}.tar.gz
+ v${_reproc_version}.zip"
 
 post_extract() {
 	# We copy the downloaded dep files to src directory so that they are found
 	# and not downloaded again by the build process. This helps to utilize caching.
-	echo "${skip_extraction}" | while read archive; do
+	mkdir -p "${wrksrc}/build/_deps"
+	for archive in ${skip_extraction}; do
 		if [ -f "${XBPS_SRCDISTDIR}/${sourcepkg}-${version}/${archive}" ]; then
-			cp "${XBPS_SRCDISTDIR}/${sourcepkg}-${version}/${archive}" \
-				"${wrksrc}/${build_wrksrc}/"
+			vsrccopy "${archive}" "${wrksrc}/build/_deps"
```

Comment:
`vsrccopy ${archive} build/_deps` should work fine, and the whole `if` check is unnecessary

Reply:
Fixed! Thanks.